frontend/lib/resolution/Resolver.cpp

```diff
@@ -337,10 +337,23 @@ Resolver::createForParentClass(Context* context,
   ret.substitutions = &substitutions;
   ret.defaultsPolicy = DefaultsPolicy::USE_DEFAULTS;
   ret.byPostorder.setupForSymbol(decl);
-  //ret.scopeResolveOnly = true;
   return ret;
 }
 
+// set up Resolver to scope resolve a parent class type expression
+Resolver
+Resolver::createForParentClassScopeResolve(Context* context,
+                                           const AggregateDecl* decl,
+                                           ResolutionResultByPostorderID& byId)
+{
```

Comment:
Hmm, that would be interesting, but I can't pass `nullptr` for the `substitutions` argument (because it is a reference). At the same time, I don't think this concern is worth refactoring these into a helper function. These functions are only a few lines long, and additionally `createForParentClassScopeResolve` is temporary (until we can fully resolve instead of just scope resolving). So, I'd like to leave this the way it is.